spiketoolkit/comparison/groundtruthcomparison.py

```diff
@@ -0,0 +1,392 @@
+from .basecomparison import BaseComparison
+
+import numpy as np
+import pandas as pd
+
+
+# Note for dev,  because of  BaseComparison internally:
+#     sorting1 = gt_sorting
+#     sorting2 = other_sorting
+
+class GroundTruthComparison(BaseComparison):
+    """
+    Class to compare a sorter to ground truth (GT)
+    
+    This class can:
+      * compute a "macth between gt_sorting and other_sorting
+      * compte th score label (TP, FN, CL, FP) for each spike
+      * count by spiketrain of GT the total of each (TP, FN, CL, FP) into a Dataframe 
+        GroundTruthComparison.count
+      * compute the confusion matrix .get_confusion_matrix()
+      * compute some performance metric with several strategy based on 
+        the count score by spiketrain
+      * count how much well detected units with some threshold selection
+      * count false positve detected units
+      * count units detected twice (or more)
+      * summary all this
+    """
+    def __init__(self, gt_sorting, other_sorting, gt_name=None, other_name=None,
```

Comment:
`other_sorting` maybe could be `output_sorting` or `test_sorting`?

Not sure :)

Reply:
Also in our case we don't have mertics taht is penalized by cl ?